locales-pending/premium.ftl

```diff
@@ -25,7 +25,7 @@ toolbar-app-picker-by-mozilla = Made by { -brand-mozilla }
 
 user-menu-trigger-label = Open user menu
 user-menu-trigger-tooltip = Profile
-user-menu-manage-fxa-label = Manage your { -brand-fx-account }
+user-menu-manage-fxa-label = Manage your { -brand-fx-account-2 }
```

Comment:
The good news is that you can change these strings without new IDS, since this file is not localized.